pages.ja/common/docker-compose.yml

```
@@ -0,0 +1,36 @@
+# docker compose
+
+> 複数コンテナを持つDockerアプリケーションの実行と管理をします。
+> 詳しくはこちら: <https://docs.docker.com/compose/reference/>.
+
+- 実行中のコンテナ全てをリスト表示する:
+
+`docker compose ps`
+
+- カレントディレクトリにある `docker-compose.yml`ファイルを使用してバックグラウンドで全てのコンテナを作成・起動する:
+
+`docker compose up -d`
+
+- 全てのコンテナを起動し、必要に応じて再ビルドする:
+
+`docker compose up --build`
+
+- 代替composeファイルを使って全てのコンテナを起動する:
+
+`docker compose --file {{ファイルパス}} up`
+
+- 全てのコンテナを停止する:
```

```suggestion
- 実行中のすべてのコンテナーを停止します:
```
Shouldn't it be something like this? (Stop all running containers)?

---

Surely！ :bow: 
Fixed in https://github.com/tldr-pages/tldr/pull/9828/commits/c0a9e330427d9d5aeaf49b322b1f4376de391a9e